Description: Define to use Boost.Filesystem version 2 to avoid FTBFS with
boost1.46, where version 3 is the default.
Author: Cristian Greco <cristian@regolo.cc>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -102,6 +102,8 @@
 	ADD_DEFINITIONS(-DNDEBUG)
 endif(CMAKE_BUILD_TYPE MATCHES Debug)
 
+ADD_DEFINITIONS(-DBOOST_FILESYSTEM_VERSION=2)
+
 CHECK_INCLUDE_FILES(sys/epoll.h HAVE_SYS_EPOLL_H)
 CHECK_FUNCTION_EXISTS(kqueue HAVE_KQUEUE)
 CONFIGURE_FILE(config.h.in config.h)
